cli/internal/cmd/init.go

```diff
@@ -208,7 +208,11 @@ func (i *initCmd) initialize(cmd *cobra.Command, newDialer func(validator atls.V
 	if err != nil {
 		return fmt.Errorf("generating measurement salt: %w", err)
 	}
-	i.log.Debugf("Measurement salt: %x", measurementSalt)
+	i.log.Debugf("Writing measurement salt: %x in idFile", measurementSalt)
+	idFile.MeasurementSalt = measurementSalt
+	if err := i.fileHandler.WriteJSON(constants.ClusterIDsFilename, &idFile, file.OptOverwrite); err != nil {
+		return fmt.Errorf("writing cluster ID file: %w", err)
+	}
```

Comment:
```suggestion
```
We already write the file to disk during `writeOutput`, I don't think it is necessary to write it again at this stage.
The measurementSalt also cannot be reused for cluster initialization, so if anything fails during init, we would have to regenerate it anyways.